### DISCUSSION TITLE
Difference between dreambooth, fine tune and native training?

### DISCUSSION THREAD

**Discussion by gilroff** [Linaqruf/kohya-trainer#160] [Q&A]
im clueless.

**Comment by Rudy34160** [comment]
Where can we have more documentation (and possibly examples) on these different types of training and their possible uses?

**Reply by Eden-Luminar** [reply]
I only know that all above trains is using one model here - I think that Dreambooth is one model here, Fine-tune is one model here, Native is no model here??? One Model is go together with relevant algorithm to method:

1. Dreambooth is one Dreambooth Model + Dreambooth Method + Dreambooth Algorithms
2. Fine-tune is is one Fine-tuned Model + Fine-tuning Method + Fine-tuning Algorithms
3. Native Training may be likely 1 or 2 but it is not important to notice, or I don't know

**Reply by Eden-Luminar** [reply]
As I know, on HuggingFace having 2 sites are Dreambooth Concept Library and LoRA Concept Library - So maybe Kohya Trainers is only use one of Dreambooth Concept Library and LoRA Concept Library - I only guess, it only answer by Kohya Git Owner

**Comment by toyssamurai** [comment]
100% agree. All I can tell so far is that given the same set of images/captions, using the Kohya LoRA Fine-Tuning method gave me a much better result than using Kohya LoRA Dreambooth. The problem is, I don't really have any idea what a LoRA Fine-Tuning is. I have a rough idea what "Dreambooth" itself is, and thus, kind of understand what LoRA Dreambooth does (and from what I know, it's possible to train a full model using Dreambooth, then extract a LoRA model out). 

Maybe this is what Kohya LoRA Fine-Tuning does? First, fine-tune a model, then extract the LoRA out?
